docs/provider/unlocker.md

```diff
@@ -0,0 +1,38 @@
+---
+title: 'Unlocker'
+sidebar_position: 4
+---
+
+The unlocker is a service coming as a binary that lets you automatically unlock your running daemon once it is initialized.
+
+This service sources the unlocking password from one of the providers listed below:
+
+- [File](#unlock-with-file)
+- AWS KMS (Coming soon)
+- Hashicorp Vault (Coming soon)
+- Kubernetes (Coming soon)
+
+Along with the password, these providers also provide a way to possibly source a TLS certificate in case the communication with the daemon needs to be encrypted.
+
+It is possible to select the preferred provider with the flag `--provider`.
```

Comment:
If can move this sentence before and document the `providerType` with the keyword for each